rmgpy/quantity.py

```diff
@@ -405,7 +405,7 @@ def __str__(self):
             uncertainty = '[{0}]'.format(','.join(uncertainty))
         
         result = '{0}'.format(value)
-        if any(self.uncertainty != 0.0):
+        if (self.uncertainty>0).any():
```

Comment:
Can you teach me what this did and what it does now?
I'm used to the format `if any([a, b, c]):`, but here it looks like we're checking whether `[a, b, c] > 0` instead of `entry >0 for entry in [a, b, c]`

Reply:
Basically this gave me an error related to comparing an array with a value whenever it was actually called.  I think it just never got called in normal operation until I was trying to log stuff.  Essentially the line is suppose to display uncertainties if they aren't all zero, but it resulted in an error until I fixed it.

Reply:
This seems odd to me. You're still comparing an array to a value, so it doesn't make sense why this would work any differently...

Reply:
Your right, but I can tell you for certain that the first one crashed everytime and the second one displays properly